config/default_configs/default_config.yml

```
@@ -232,6 +232,9 @@ ls_adv:
 external_forcing:
   help: "External forcing for single column experiments [`nothing` (default), `GCM`]"
   value: ~
+external_forcing_file:
+  help: "External forcing file containing large-scale forcings, initial conditions, and boundary conditions [`nothing` (default), `GCM`]"
```

```suggestion
  help: "External forcing file containing large-scale forcings, initial conditions, and boundary conditions [`nothing` (default), `path/to/file`]"
```
something like this?